cli/cmd/token/revoke.go

```diff
@@ -7,26 +7,67 @@ import (
 )
 
 func RevokeCmd(ch *cmdutil.Helper) *cobra.Command {
+	var all bool
+
 	revokeCmd := &cobra.Command{
-		Use:   "revoke <token-id>",
-		Args:  cobra.ExactArgs(1),
-		Short: "Revoke personal access token",
+		Use:   "revoke [token-id]",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Revoke personal access token(s)",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			client, err := ch.Client()
 			if err != nil {
 				return err
 			}
 
+			if len(args) > 0 && all {
+				ch.PrintfWarn("Cannot specify a token ID when using --all flag\n")
+				return cmd.Usage()
```

Comment:
Better to return errors so we can handle them centrally in Cobra's middleware (for consistent formatting of fatal errors):
```suggestion
				return fmt.Errorf("cannot specify a token ID when using --all flag")
```